docker-app/qfieldcloud/settings.py

```diff
@@ -629,4 +629,9 @@ def before_send(event, hint):
     "copyright": "OPENGIS.ch",
     # Additional custom CSS file for the Django Admin pages.
     "custom_css": "css/admin.css",
+    "show_ui_builder": True,
```

Comment:
Shall we keep this? Or make it a setting for example?

Reply:
`show_ui_builder` is more about letting us devs test settings that we want to put in `settings.py`, definitely not to be kept.

Anyway, to be discussed in the sprint meeting ;-)